models/parser/TranslationPurifier.php

```diff
@@ -17,6 +17,7 @@ public static function configure( $config)
         $config->set('HTML.Doctype', 'HTML 4.01 Transitional');
         $def =  $config->getHTMLDefinition(true);
         $def->addAttribute('a', 'href', new ParameterURIDef());
+        $def->addAttribute('a', 'target', new ParameterURIDef());
```

Comment:
Is `ParameterURIDef` really the right type? Think this is just for urls. I've used `Text` in the past [here ](http://htmlpurifier.org/docs/enduser-customize.html) they suggest `$def->addAttribute('a', 'target', 'Enum#_blank,_self,_target,_top');`

Reply:
@buddh4 yes, my first commit was not fully correct, thanks for the doc url, I used this `new HTMLPurifier_AttrDef_Enum(['_blank', '_self', '_parent', '_top'])` from there in new commit.

Reply:
We should probably use this also in the core and other places.